main.py

```diff
@@ -0,0 +1,74 @@
+from fastapi import FastAPI, Depends, Query, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+from models import Book
+from schemas import BookListResponse
```

Comment:
Ensure that `BookListResponse` is correctly defined in the `schemas` module. It should match the expected response structure for the `/books/` endpoint.